about.html

```diff
@@ -43,7 +40,7 @@
         <li><i class="fab fa-twitter light-grey upperside-item"></i></li>
         <li><a href="#" class="logout light-grey upperside-item">Logout</a></li>
         <li><a href="#" class="mypage light-grey upperside-item">Mypage</a></li>
-        <li><a href="#" class="korean light-grey upperside-item">English</a></li>
+        <li><a href="#" class="English light-grey upperside-item">English</a></li>
```

Comment:
- [Optional] Kindly you for the font style and color are not the same and you have loosed margin and padding from the mobile version, Kindly stick to the given design as much as possible to have a better [design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 👍👍

https://user-images.githubusercontent.com/52010789/217487160-d820c983-3d17-4734-a9e9-a3c46d664cbf.mp4